assets/xml/objects/object_an1.xml

```diff
@@ -1,88 +1,107 @@
-        <Limb Name="object_an1_Standardlimb_0124FC" Type="Standard" EnumName="OBJECT_AN1_LIMB_04" Offset="0x124FC" />
-        <Limb Name="object_an1_Standardlimb_012508" Type="Standard" EnumName="OBJECT_AN1_LIMB_05" Offset="0x12508" />
-        <Limb Name="object_an1_Standardlimb_012514" Type="Standard" EnumName="OBJECT_AN1_LIMB_06" Offset="0x12514" />
-        <Limb Name="object_an1_Standardlimb_012520" Type="Standard" EnumName="OBJECT_AN1_LIMB_07" Offset="0x12520" />
-        <Limb Name="object_an1_Standardlimb_01252C" Type="Standard" EnumName="OBJECT_AN1_LIMB_08" Offset="0x1252C" />
-        <Limb Name="object_an1_Standardlimb_012538" Type="Standard" EnumName="OBJECT_AN1_LIMB_09" Offset="0x12538" />
-        <Limb Name="object_an1_Standardlimb_012544" Type="Standard" EnumName="OBJECT_AN1_LIMB_0A" Offset="0x12544" />
-        <Limb Name="object_an1_Standardlimb_012550" Type="Standard" EnumName="OBJECT_AN1_LIMB_0B" Offset="0x12550" />
-        <Limb Name="object_an1_Standardlimb_01255C" Type="Standard" EnumName="OBJECT_AN1_LIMB_0C" Offset="0x1255C" />
-        <Limb Name="object_an1_Standardlimb_012568" Type="Standard" EnumName="OBJECT_AN1_LIMB_0D" Offset="0x12568" />
-        <Limb Name="object_an1_Standardlimb_012574" Type="Standard" EnumName="OBJECT_AN1_LIMB_0E" Offset="0x12574" />
-        <Limb Name="object_an1_Standardlimb_012580" Type="Standard" EnumName="OBJECT_AN1_LIMB_0F" Offset="0x12580" />
-        <Limb Name="object_an1_Standardlimb_01258C" Type="Standard" EnumName="OBJECT_AN1_LIMB_10" Offset="0x1258C" />
-        <Limb Name="object_an1_Standardlimb_012598" Type="Standard" EnumName="OBJECT_AN1_LIMB_11" Offset="0x12598" />
-        <Limb Name="object_an1_Standardlimb_0125A4" Type="Standard" EnumName="OBJECT_AN1_LIMB_12" Offset="0x125A4" />
-        <Limb Name="object_an1_Standardlimb_0125B0" Type="Standard" EnumName="OBJECT_AN1_LIMB_13" Offset="0x125B0" />
-        <Limb Name="object_an1_Standardlimb_0125BC" Type="Standard" EnumName="OBJECT_AN1_LIMB_14" Offset="0x125BC" />
-        <Skeleton Name="object_an1_Skel_012618" Type="Flex" LimbType="Standard" LimbNone="OBJECT_AN1_LIMB_NONE" LimbMax="OBJECT_AN1_LIMB_MAX" EnumName="ObjectAn1Limb" Offset="0x12618" />
-        <Animation Name="object_an1_Anim_013048" Offset="0x13048" /> <!-- Original name is "an_walk" -->
-        <Animation Name="object_an1_Anim_013E1C" Offset="0x13E1C" /> <!-- Original name is "anj_tetunagu_loop" ("joined hands") -->
+
+        <!-- Chopsticks used for cooking -->
+        <DList Name="gAnju1ChopsticksDL" Offset="0x12478" /> <!-- Original name is "an_hashi_model" ("chopsticks​") -->
+
+        <!-- Skeleton's limbs -->
+        <Limb Name="gAnju1RootLimb" Type="Standard" EnumName="ANJU1_LIMB_ROOT" Offset="0x124D8" />
+        <Limb Name="gAnju1TorsoLimb" Type="Standard" EnumName="ANJU1_LIMB_TORSO" Offset="0x124E4" />
+        <Limb Name="gAnju1LeftUpperArmLimb" Type="Standard" EnumName="ANJU1_LIMB_LEFT_UPPERARM" Offset="0x124F0" />
```

Comment:
When "upper arm" is written in an `EnumName`, it is (currently) exclusively written as `UPPER_ARM`. Could you change them for consistency?